osu.Game/Overlays/Settings/SettingsSection.cs

```diff
@@ -83,53 +84,58 @@ protected SettingsSection()
         [BackgroundDependencyLoader]
         private void load(OverlayColourProvider colourProvider)
         {
-            AddRangeInternal(new Drawable[]
+            AddInternal(new OsuContextMenuContainer
```

Comment:
Shitting these all over the code base is pretty bad.

This one should be at a `SettingsOverlay` level if anything, IMHO.

Also it should have a comment mentioning what it's being used by or something. Or it should just be moved to global because this is ugly.